addons/website/tests/test_views.py

```diff
@@ -264,7 +264,7 @@ def test_field_tail(self):
         )
 
 
-@tagged('-at_install', 'post_install')
+@tagged('-at_install', 'post_install', 'JPP')
```

Comment:
🤨

Reply:
thanks, fixed in https://github.com/odoo/odoo/pull/47310